File: src/main/java/io/codehive/bloc4j/Main.java
package io.codehive.bloc4j;

import io.codehive.bloc4j.lib.Window;
import io.codehive.bloc4j.math.Vec2i;

public class Main {

  public static void main(String[] args) {
    MainApplication app = new MainApplication();

    Window window = new Window(app, "Bloc4J", new Vec2i(800, 600));

    window.init();
  }

}

File: src/main/kotlin/io/codehive/bloc4j/Main.kt
package io.codehive.bloc4j

import io.codehive.bloc4j.graphics.MainApplication
import io.codehive.bloc4j.graphics.lib.Window
import org.joml.Vector2i

fun main() {
  val application = MainApplication()

  Window.init(application, "Bloc4J", Vector2i(800, 600))
}
